docs/ref/models/options.txt

```diff
@@ -24,17 +24,12 @@ Available ``Meta`` options
 
 .. attribute:: Options.app_label
 
-    If a model exists outside of the standard locations (:file:`models.py` or
-    a ``models`` package in an app), the model must define which app it is part
-    of::
+    If a model exists outside of an application in :setting:`INSTALLED_APPS` or
```

Comment:
@aaugustin -- I took this documentation from the current deprecation warning (which should be changed to error in 1.9). Does it seem okay? I guess it might change based on the outcome of #21680 and #21682.